web/src/main/webapp/xslt/base-layout-cssjs-loader.xsl

```diff
@@ -246,13 +246,6 @@
     <xsl:variable name="appConfig"
                   select="util:getUiConfiguration(/root/request/ui)"/>
 
-    <xsl:if test="$angularApp = 'gn_search'">
-      <script src="{$uiResourcesPath}lib/d3_timeseries/d3.min.js?v={$buildNumber}"></script>
-      <script src="{$uiResourcesPath}lib/timeline/timeline-zoomable.js?v={$buildNumber}"></script>
-      <link rel="stylesheet" href="{$uiResourcesPath}lib/timeline/timeline.css"/>
-      <link rel="stylesheet" href="{$uiResourcesPath}lib/d3_timeseries/nv.d3.min.css"/>
-    </xsl:if>
```

Comment:
Can be removed the related javascript / css files or are used in other places?

Reply:
They are imported just above line 241 - probably a wrong merge. So they are duplicated